README.md

```diff
@@ -53,7 +53,7 @@ You can now run this API with:
 
 From an LmP device:
 ~~~
- $ DEVICE_API=http://<IP of docker-compose host>/sign lmp-device-register [-T <TOKEN>]
+ $ sudo DEVICE_API=http://<IP of docker-compose host>/sign lmp-device-register [-T <TOKEN>] -f <factory>
```

Comment:
IIUC the factory argument is expected to go from the `/etc/os-release` in the normal process.

As for the `sudo`, I'm not sure yet - I'd rather ask @ldts to comment on it.

Reply:
Just to clarify the motivation.
By running the command without sudo it fails with `Error locking memory` (I am testing using QEMU)

Reply:
You have to be root. There's probably an assumption in almost all our docs that commands like that require root.

As per `-f <factory>` that will be required for people using the public LMP build (ie. the fast first target).

Reply:
Then from what @doanac I can deduct that maybe we should go with:
```
DEVICE_API=http://<IP of docker-compose host>/sign lmp-device-register [-T <TOKEN>] [-f <factory>]
```

Reply:
i'm indifferent about sudo vs not showing sudo. If someone is really passionate about one or the other - go with that.

Reply:
I would prefer to keep the sudo.
I do not run commands as sudo by default

Also, when you run it without sudo the issue raised was not something like `"you do not have enough permissions"` so is a bit far away to assume that the error `Error locking memory` is caused by you calling the bin with the endpoint of the API without been root.

Reply:
I have no strong opinion on this.

Just ask customer support guys like @vanmaegima @angolini what they would like to see here.
This is an advice the end user will follow.

Reply:
I'd rather tell smth like `Run as root from LmP device`.  IIUC this is what the majority of customers do annyway.

Reply:
@vkhoroz ,

It was changed for `Run as root from LmP device`.
Can we get your LGTM to get this one merged?